Detector/DetComponents/CMakeLists.txt

```diff
@@ -16,10 +16,16 @@ find_package(DD4hep COMPONENTS DDG4 DDSegmentation DDRec REQUIRED)
 
 find_package(Geant4)
 
+find_package(ACTS COMPONENTS Core TGeoPlugin DD4hepPlugin Examples)
+#workaround for the lone -lGeom flag exported by ACTS
+link_directories(${ROOT_INCLUDE_DIR}/../lib)
```

Comment:
Yes, also still needed. Do we want to get rid of this workaround? Then I can open a ticket.